src/containers/ BooksForm.js

```diff
@@ -1,17 +1,66 @@
-const BooksForm = () => {
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { addBook } from '../actions ';
+
+const BooksForm = props => {
+  const { addBook } = props;
```

Comment:
### \[OPTIONAL\] Stretch Changes
- [ ] Can these props get destructured from the function parameters in this functional component to avoid additional variable declarations like in our case? What we have makes us use extra lines of code and makes the component more verbose`
- See what I have in mind below 👇 

```suggestion
// This way it still achieves the same result but with less code in a more reusable way
const BooksForm = ({ addBook }) => {
```